samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -36,56 +33,142 @@ static void sensor_cli_data_cb(struct bt_mesh_sensor_cli *cli,
 	}
 }
 
-static void sensor_cli_series_entry_cb(
-	struct bt_mesh_sensor_cli *cli, struct bt_mesh_msg_ctx *ctx,
-	const struct bt_mesh_sensor_type *sensor, uint8_t index, uint8_t count,
-	const struct bt_mesh_sensor_series_entry *entry)
+static void sensor_cli_series_entry_cb(struct bt_mesh_sensor_cli *cli, struct bt_mesh_msg_ctx *ctx,
+				       const struct bt_mesh_sensor_type *sensor, uint8_t index,
+				       uint8_t count,
+				       const struct bt_mesh_sensor_series_entry *entry)
 {
-	printk("Relative runtime in %d to %d degrees: %s percent\n",
-	       entry->value[1].val1, entry->value[2].val1,
-	       bt_mesh_sensor_ch_str(&entry->value[0]));
+	printk("Relative runtime in %d to %d degrees: %s percent\n", entry->value[1].val1,
+	       entry->value[2].val1, bt_mesh_sensor_ch_str(&entry->value[0]));
+}
+
+static void sensor_cli_setting_status_cb(struct bt_mesh_sensor_cli *cli,
+					 struct bt_mesh_msg_ctx *ctx,
+					 const struct bt_mesh_sensor_type *sensor,
+					 const struct bt_mesh_sensor_setting_status *setting)
+{
+	printk("Sensor ID: 0x%04x, Setting ID: 0x%04x\n", sensor->id, setting->type->id);
+	for (int chan = 0; chan < setting->type->channel_count; chan++) {
+		printk("\tChannel %d value: %s\n", chan,
+		       bt_mesh_sensor_ch_str(&(setting->value[chan])));
+	}
+}
+
+static void sensor_cli_desc_cb(struct bt_mesh_sensor_cli *cli, struct bt_mesh_msg_ctx *ctx,
+			       const struct bt_mesh_sensor_info *sensor)
+{
+	printk("Descriptor of sensor with ID 0x%04x:\n", sensor->id);
+	printk("\ttolerance: { positive: %s, negative: %s }\n",
+	       bt_mesh_sensor_ch_str(&sensor->descriptor.tolerance.positive),
+	       bt_mesh_sensor_ch_str(&sensor->descriptor.tolerance.negative));
+	printk("\tsampling type: %d\n", sensor->descriptor.sampling_type);
+	printk("\tperiod: %lld\n", sensor->descriptor.period);
```

Comment:
The period and update interval is not defined in the descriptor (the descriptor type mentions these as optional), perhaps remove these prints.